src/pages/Cart/Cart.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useState } from 'react';
+import { useAppContext } from '../../context/AppContext';
+import { CartItem } from '../../components/CartItem.tsx';
+import { ProductInCart } from '../../types/ProductInCart';
+import { Modal } from '../../components/Modal';
+import { ButtonBack } from '../../components/ButtonBack';
+import './cart.scss';
+
+type Props = {
+  products: ProductInCart[];
+};
+
+export const Cart: FC<Props> = ({ products }) => {
+  const { inCartCount } = useAppContext();
+  const [isOpenModal, setIsOpenModal] = useState(false);
+
+  const totalPrice = () => {
+    return products.reduce(
+      (acc: number, el: ProductInCart) => acc + el.price * el.count,
+      0,
+    );
+  };
+
+  const totalSum = totalPrice();
+
+  return (
+    <div className="cart">
+      <div className="cart__container">
+        <div className="cart__navigate">
+          {' '}
+          <ButtonBack />
```

Comment:
Why you need that? If you need add space - use `&nbsp;`
```suggestion
          &nbsp;<ButtonBack />
```